cassovary-core/src/test/scala/com/twitter/cassovary/algorithms/centrality/DegreeCentralitySpec.scala

```diff
@@ -0,0 +1,38 @@
+package com.twitter.cassovary.algorithms.centrality
+
+import com.twitter.cassovary.graph._
+import org.scalatest.{Matchers, WordSpec}
+
+/**
+ * Created by bmckown on 1/28/15.
+ */
+class DegreeCentralitySpec extends WordSpec with Matchers {
+
+  "Degree Centrality" should {
+
+    lazy val graph  = TestGraphs.g6WithEmptyNodes
+
+    lazy val unnormalizedInDegreeCentrality = Array(0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 2.0, 3.0, 1.0, 3.0, 1.0)
+    lazy val unnormalizedOutDegreeCentrality = Array(0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 3.0, 2.0, 1.0, 2.0, 1.0, 2.0)
```

Comment:
would be clearer if we have only proper values in the arrays (from index 11) and check only them.

Leading zeros may be a bit strange.